client/components/Profile.jsx

```diff
@@ -78,8 +109,8 @@ class Profile extends React.Component {
 
   handleSubmit (e) {
     const {id, updateProfile} = this.props
-    const {username, currentPassword, newPassword, confirm} = this.state
-    updateProfile(id, username, currentPassword, newPassword, confirm)
+    const {username, email, cohort, profilePic, bio, currentPassword, newPassword, confirm} = this.state
+    updateProfile(id, username, email, cohort, profilePic, bio, currentPassword, newPassword, confirm)
```

Comment:
Let's pass an object with multile properties into `updateProfile` instead of all these input parameters.